include/fastrtps/serializer/serializer.h

```diff
@@ -0,0 +1,502 @@
+#include <fastrtps/types/DynamicTypePtr.h>
+#include <fastrtps/types/TypeObjectHashId.h>
+#include <fastrtps/types/TypeObject.h>
+namespace eprosima{
+
+namespace fastcdr{
+    class Cdr ;
+}
+namespace fastrtps{
+
+namespace types{
+    class DynamicData ;
+}
+
+namespace serializer{
+
+/*********************
+ *     SERIALIZER    *
+ *********************/
+	
+template<class Serializable>
+class Serializer{
+private:
+    Serializer() = delete ;
+public:
+    Serializer<Serializable>(Serializable *p):p_(p){}
+    
+    void serialize(eprosima::fastcdr::Cdr& cdr)
+				{
```

Comment:
Remove tabs, use 4 spaces.
You can automatize replacing tabs with 4 spaces and removing trailing spaces at save time in your favorite editor.

Reply:
Look for tabs in the complete PR and replace them with 4 spaces.

Reply:
done